source/_integrations/tilt_pi.markdown

```diff
@@ -0,0 +1,101 @@
+---
+title: Tilt Pi
+description: Instructions on how to configure a Tilt Pi.
+ha_category:
+  - Sensor
+ha_release: '2025.3'
+ha_iot_class: Local Polling
+ha_quality_scale: Bronze
+ha_config_flow: true
+ha_codeowners:
+  - '@michaelheyman'
+ha_domain: tilt_pi
+related:
+  - url: https://tilthydrometer.com/collections/tilt-pi
+    title: Tilt Pi product page
+---
+
+This integration allows you to connect a Tilt Pi to Home Assistant, which will
+automatically discover Tilt Hydrometer devices.
+
+The Tilt Pi is a Raspberry Pi-based device that can read data from Tilt
+Hydrometers and broadcast it over the network. The Tilt Pi can be used to
+monitor the fermentation of beer, wine, cider, and other beverages.
+
+The benefit of the Tilt Pi is that it can be placed in a location with better
+reception than the Tilt Hydrometer itself, allowing for more reliable data
+collection.
+
+{% note %}
+For instances where you only have a Tilt Hydrometer and not a Tilt Pi, consider
+using the [Tilt Hydrometer integration](/integrations/tilt_ble).
+{% endnote %}
+
+## Prerequisites
+
+1. Have a Raspberry Pi with the Tilt Pi software installed.
+2. Have a Tilt Hydrometer in range of the Tilt Pi.
+3. Have the IP address or hostname of the Tilt Pi.
+
+{% include integrations/config_flow.md %}
+{% configuration_basic %}
+Name:
+    description: "The friendly name for the Tilt Pi."
+    required: true
+    type: string
```

Comment:
```suggestion
```
the config_basic block doesn't show these
![image](https://github.com/user-attachments/assets/1cf2c4f8-8b66-40b8-8370-69133a191e06)